python_ta/checkers/unnecessary_f_string_checker.py

```diff
@@ -0,0 +1,39 @@
+"""Checker to check for unncessary f-strings that only consist of unmodified variables"""
+
+from __future__ import annotations
+
+from astroid import nodes
+from pylint.checkers import BaseChecker
+from pylint.checkers.utils import only_required_for_messages
+from pylint.lint import PyLinter
+
+
+class FormattedStringChecker(BaseChecker):
+    """A checker class that reports unnecessary uses of f-strings when they can be replaced
+    with the variable directly"""
+
+    name = "unnecessary-f-string"
+    msgs = {
+        "E9920": (
+            'Unnecessary use of f-strings in the expression "f{%s}". Use "str(%s)" instead.',
```

Comment:
"f-strings" -> "an f-string"

For the code parts, let's use backticks instead of quotes.

Finally, instead of `f{%s}` you should use `f"{%s}"`.